diff --git a/fontawesome.module b/fontawesome.module
index d5c5f1a..801df4d 100644
--- a/fontawesome.module
+++ b/fontawesome.module
@@ -17,7 +17,7 @@ use Drupal\Core\TypedData\Plugin\DataType\ItemList;
 function fontawesome_help($route_name, RouteMatchInterface $route_match) {
   switch ($route_name) {
     case 'help.page.fontawesome':
-      return '<p><i class="far fa-font-awesome fa-2x"></i> ' . t('<a href=":fontawesome_url">Font Awesome</a> is an iconic font and CSS toolkit. Font Awesome gives you scalable vector icons that can instantly be customized — size, color, drop shadow, and anything that can be done with the power of CSS. For more information on how to use Font Awesome, see the <a href=":fontawesome_examples_page">Font Awesome Examples page</a>.', [
+      return '<p><i class="far fa-font-awesome fa-2x"></i> ' . t('<a href=":fontawesome_url">Font OSOM</a> is an iconic font and CSS toolkit. Font Awesome gives you scalable vector icons that can instantly be customized — size, color, drop shadow, and anything that can be done with the power of CSS. For more information on how to use Font Awesome, see the <a href=":fontawesome_examples_page">Font Awesome Examples page</a>.', [
         ':fontawesome_url' => 'https://fontawesome.com',
         ':fontawesome_examples_page' => 'https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use',
       ]) . '</p>';
